examples/ddp_00.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass, field
+import logging
+import os
+from typing import Any, List
+
+import hydra
+from hydra.core.config_store import ConfigStore
+from omegaconf import DictConfig, MISSING
+import torch
+import torch.distributed as dist
+
+log = logging.getLogger(__name__)
+
+
+defaults = [
+    {"hydra/launcher": "joblib"}
```

Comment:
```suggestion
    {"override hydra/launcher": "joblib"}
```

I believe this will be required moving forward, correct @omry ?

Reply:
Yes.

Reply:
As of Hydra 1.1, which we are now official shooting for.